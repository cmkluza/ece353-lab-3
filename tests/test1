./sim-mips -b 1 1 1 ./tests/asm/test1.s ./tests/out/test1.out