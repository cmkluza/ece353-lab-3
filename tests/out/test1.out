program name: ./tests/asm/test1.s
stage utilization: 0.538462  0.461538  0.461538  0.000000  0.461538 
register values 0  0  0  0  0  0  0  15  10  0  0  0  0  0  0  20  7  0  0  0  0  0  0  0  0  0  0  0  5  0  8  28
